package.json

```diff
@@ -45,6 +45,7 @@
     "@types/graceful-fs": "^4.1.5",
     "@types/micromatch": "^4.0.1",
     "@types/node": "^14.14.37",
+    "@vercel/edge-config": "^0.4.0",
```

Comment:
This needs to be added to the lock file too. You might need to run in linux x64 to update since some of the dependencies don't work on arm unfortunately.

Reply:
Done